lxd/instance_snapshot.go

```diff
@@ -776,10 +779,12 @@ func snapshotDelete(s *state.State, r *http.Request, snapInst instance.Instance)
 		return snapInst.Delete(false)
 	}
 
-	parentName, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
+	snapName := snapInst.Name()
```

Comment:
Why didn't you keep the `snapInst.Name()` directly in the `pi.GetParentAndSnapshotName` argument, rather than assign it to its own variable and then immediately replace the variable with the output of `api.GetParentAndSnapshotName`, seems a bit confusing to me as the reader?